Gemfile

```diff
@@ -26,6 +28,8 @@ gem 'jbuilder', '~> 2.5'
 # gem 'redis', '~> 4.0'
```

Comment:
Remove all the comments from your Gemfile(s), that way the file is simple and focused. 
If you want to add comments, use comments to group similar/related Gems eg

```
#///////////// Authentication && Authorization ////////////////////////////

gem 'jwt', '~> 2.1'
gem 'devise', '~> 4.4', '>= 4.4.1'

#///////////// JSON SERIALIZATION /////////////////////////////////////////

gem 'active_model_serializers', '~> 0.10.7'

#///////////// File Uploading /////////////////////////////////////////////

gem 'aws-sdk-s3', '~> 1.8', '>= 1.8.2'
```